apiary.apib

```diff
@@ -718,6 +721,22 @@ Cancel a user invite.
 
 + Response 204
 
+### userInvites.resend [GET /userInvites.resend]
+
+Resend a user invite.
```

Comment:
Should we add more information about what this does? That it cancels the previous invitation and creates a new one?